app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def cope_file(command: str) -> any:
+    new_command = command.split()
+    if new_command[1] == new_command[2]:
```

Comment:
Please add an additional check to ensure that the length of new_command is 3 and that the zeroth element is 'cp'.